boards/common/nrf52xxxdk/include/periph_conf.h

```diff
@@ -108,6 +108,17 @@ static const i2c_conf_t i2c_config[] = {
 #define I2C_NUMOF           (sizeof(i2c_config) / sizeof(i2c_config[0]))
 /** @} */
 
+/**
+ * @name   PWM configuration
+ * @{
+ */
+static const pwm_conf_t pwm_config[] = {
+    { NRF_PWM0, { 23, 24, 25, 26 } }
```

Comment:
P0.23 doesn't seem to work here. Does it on your board?

Reply:
@PeterKietzmann Yes, it works for me on nrf52dk. Is it possible, that you are testing a different board with P0.23 already in use by another module?

Reply:
To be honest, I don't remember. **But** in my opinion any board that provides the PWM feature and makes use of this *periph_conf.h* should work with the provided configuration of **at least** be documented.

Reply:
I double check that, but i am confident 23 works for me, because i mostly used that pin for debugging.

Reply:
Would be great if you could check the manuals of the other nrf52 boards that we support. I think it's just 2-3...